packages/block-library/src/embed/styles.native.scss

```diff
@@ -25,6 +25,7 @@
 	justify-content: center;
 	align-items: center;
 	fill: $gray-dark;
+	margin-bottom: 6;
```

Comment:
The `embed__icon` style is used in other places so it would be nice to check how this margin value might affect.